src/dashboard/src/components/administration/views.py

```diff
@@ -456,11 +457,14 @@ def general(request):
     dashboard_uuid = helpers.get_setting('dashboard_uuid')
     try:
         pipeline = storage_service._get_pipeline(dashboard_uuid)
-    except Exception:
-        messages.warning(request, _("Storage server inaccessible. Please contact an administrator or update storage service URL below."))
-    else:
-        if not pipeline:
-            messages.warning(request, _("This pipeline is not registered with the storage service or has been disabled in the storage service. Please contact an administrator."))
+    except Exception, err:
```

Comment:
You need to add some logic to make sure that the `'storage_service_use_default_config'` key of `initial_data` is a boolean. Right now it's a string and not a boolean because dashboard settings values are untyped. This is the same issue addressed by https://github.com/artefactual/archivematica/pull/847. Using that approach, I would add a line like:

    initial_data['storage_service_use_default_config'] = {
        'False': False}.get(
            initial_data.get('storage_service_use_default_config', True),
            True)